framework/console/controllers/BaseMigrateController.php

```diff
@@ -465,11 +493,35 @@ public function actionCreate($name)
             throw new Exception("The migration name should contain letters, digits and/or underscore characters only.");
         }
 
-        $name = 'm' . gmdate('ymd_His') . '_' . $name;
-        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $name . '.php';
+        $className = 'm' . gmdate('ymd_His') . '_' . $name;
+        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $className . '.php';
 
         if ($this->confirm("Create new migration '$file'?")) {
-            $content = $this->renderFile(Yii::getAlias($this->templateFile), ['className' => $name]);
+            if (preg_match('/^Create(.+)$/', $name, $matches)) {
+                $content = $this->renderFile (Yii::getAlias ($this->templateFileGenerators['create']), [
+                    'className' => $className,
+                    'table' => strtolower($matches[1]),
```

Comment:
Table names could be non-ANSI as far as I remember. Better to use mb_ variant of string functions.